GVRf/Framework/framework/src/main/java/org/gearvrf/GVRComponent.java

```diff
@@ -109,17 +121,40 @@ public boolean hasOwnerObject() {
     }
     
     /**
+     * Enable or disable this component.
+     * @param flag true to enable, false to disable.
+     * @see enable
+     * @see disable
+     * @see isEnabled
+     */
+    public void setEnable(boolean flag) {
```

Comment:
Can this be a private method? Seems most places settled on enable/disable public APIs.